mllib/src/main/scala/org/apache/spark/ml/regression/AFTSurvivalRegression.scala

```diff
@@ -276,6 +280,7 @@ class AFTSurvivalRegression @Since("1.6.0") (@Since("1.6.0") override val uid: S
     val intercept = parameters(1)
     val scale = math.exp(parameters(0))
     val model = new AFTSurvivalRegressionModel(uid, coefficients, intercept, scale)
+    instr.logSuccess(model)
```

Comment:
Also, would you mind changing the doc in `Instrumentation.logSuccess` from:

"Logs the successful completion of the training session and the value of the learned model." 

to

"Logs the successful completion of the training session."

Since we currently don't do anything other than log a string inside `logSuccess`.